static/js/util/profile_edit.js

```diff
@@ -159,6 +162,7 @@ export function boundDateField(keySet: string[], label: string, omitDay: boolean
   let onBlur = () => {
     updateValidationVisibility(keySet);
     updateProfileValidation(profile, validator);
+    sendFormFieldEvent(keySet);
```

Comment:
I'm more interested in how useful the test is than in the coverage number, but I agree it would be good to add them here.